bodhi/tests/client/test___init__.py

```diff
@@ -1597,7 +1597,7 @@ def test_edit_with_errors(self, send_request):
         result = runner.invoke(
             client.edit_release,
             ['--name', 'F27', '--long-name', 'Fedora 27, the Greatest Fedora!', '--url',
-             'http://localhost:6543', '--username', 'bowlofeggs', '--password', 's3kr3t'])
+             'http://localhost:6543', '--user', 'bowlofeggs', '--password', 's3kr3t'])
 
         self.assertEqual(result.exit_code, 1)
         self.assertEqual(result.output, ("ERROR: an error was encountered... :(\n"))
```

Comment:
We should add a test to make sure the old ```bodhi-manage-releases``` CLI still uses ```--username``` for edit and create.

Reply:
What could be a good way to test this use case? I am thinking of two ways currently:

1. Modify the ```cli``` object used in ```bodhi releases``` and call that using ```--username```.

2. Call ```bodhi-manage-releases``` by accessing terminal from test code.

Reply:
I think the click runner might still work, similarly to how it is used in the existing tests. If this doesn't work, we can just use mock to ensure the expected changes are made to the cli object by ```main()```. I don't think you want to run it in a shell because it will not be possible to mock and it will talk to a live server.

Reply:
The click runner requires a method to invoke. However, we don't have a method for ```bodhi-manage-releases create``` as we just forward it to ```bodhi releases create```. So I will probably go with using mock to ensure expected changes.